tests/platform_tests/link_flap/test_cont_link_flap.py

```diff
@@ -150,32 +164,41 @@ def test_cont_link_flap(self, request, duthosts, nbrhosts, enum_rand_one_per_hws
 
         # Check the FRR daemons memory usage at end
         end_time_frr_daemon_memory = {}
-        incr_frr_daemon_memory_threshold = {}
+        incr_frr_daemon_memory_threshold = defaultdict(lambda: [])
+
         for daemon in frr_demons_to_check:
-            incr_frr_daemon_memory_threshold[daemon] = 10 if tbinfo["topo"]["type"] in ["m0", "mx"] else 5
-            min_threshold_percent = 1 / float(start_time_frr_daemon_memory[daemon]) * 100
-            if min_threshold_percent > incr_frr_daemon_memory_threshold[daemon]:
-                incr_frr_daemon_memory_threshold[daemon] = math.ceil(min_threshold_percent)
-            logging.info(f"The memory increment threshold for frr daemon {daemon} "
-                         f"is {incr_frr_daemon_memory_threshold[daemon]}%")
+            for asic_index, asic_frr_memory in enumerate(start_time_frr_daemon_memory[daemon]):
```

Comment:
Hey @auspham, I think here `asic_index` (starting with 0 and then 1, 2, 3 etc) is actually from iterating `duthost.asics`, where `asics` is from `asics_present`. Can we confirm that the `asics_present` is always 0,1,2,3,etc? Would it be 0 and then 4, 6 etc?  Btw, if you want to know the actual asic index, you can call `asic.asic_index` when iterating `duthost.asics`.

Reply:
Good pick, this is indeed from asics_present. Upons debugging it just happen to be 0,1,2 on the device i'm running. I'll make this change